### ISSUE TITLE
Clicking on the quick fix light bulb doesn't show the context menu

### ISSUE THREAD

**Issue by rcjsuen** [Microsoft/monaco-editor#574]
**monaco-editor version:** [Monaco Editor Playground](https://microsoft.github.io/monaco-editor/playground.html)

**Browser:**
Google Chrome 60.0.3112.113 (Official Build)
Microsoft Edge 38.14393.1066.0 EdgeHTML 14.14393

**OS:** Windows 10

1. Open the Monaco Editor Playground.
2. Replace the JavaScript with the following snippet.
```JavaScript
monaco.languages.registerCodeActionProvider("json", {
    provideCodeActions: function(model, range, context, token) {
        return [
            {
                command: {
                    id: "command.id",
                    title: "command title"
                },
                score: 0
            }
        ]
    }
});

monaco.editor.create(document.getElementById("container"), {
	value: "{ \"key\": \"value\" }",
	language: "json",
    glyphMargin: true
});
```
3. Run the code.
4. The light bulb should appear. If not, click inside the JSON string. The light bulb should then appear if it's not already visible.
5. Click on the light bulb, no context menu will appear.
6. Hit <kbd>CTRL</kbd> + <kbd>.</kbd> to get the command to popup. The menu will appear but there the menu item has no text in it.

**Comment by akosyakov** [comment]
@alexandrudima is there a way to fix it and make 0.10.1? It seems to be regression and was working fine in 0.8.x.

**Comment by akosyakov** [comment]
I want to look into it. @jrieken Will it be fine? Could you give me some code pointers?

**Comment by jrieken** [comment]
Hmmm.... start by looking for implementors of the context menu service

**Comment by akosyakov** [comment]
The following works with vscode master:
```ts
monaco.languages.registerCodeActionProvider("json", {
    provideCodeActions: function(model, range, context, token) {
        return [
			{
				id: "command.id",
				title: "command title"
			}
        ]
    }
});

const editor = monaco.editor.create(document.getElementById("container"), {
	value: "{ \"key\": \"value\" }",
	language: "json",
	lightbulb: {
		enabled: true
	}
});
```

I had to enable lightbulbs explicitly and they are rendered inline not as a glyph, but clicking on a lightbulb shows the context menu.

**Comment by jrieken** [comment]
> I had to enable lightbulbs explicitly and they are rendered inline not as a glyph, but clicking on a lightbulb shows the context menu.

Then there is no issue, right? Do want to change the default?

**Comment by rcjsuen** [comment]
@jrieken Well, according to an [earlier comment](https://github.com/Microsoft/monaco-editor/issues/574#issuecomment-326567070), it was working fine in 0.8.x.

Are you saying that it shouldn't have worked in 0.8.x? Are perhaps the documentation is inadequate somewhere?

**Comment by jrieken** [comment]
That's probably when we have added the option to show/hide the light bulb.

**Comment by akosyakov** [comment]
@rcjsuen it looks like it was fixed in the meantime and should work again in next version.

**Comment by rcjsuen** [comment]
@alexandrudima Was this supposed to be closed by Microsoft/vscode#41927?

**Comment by alexandrudima** [comment]
... sometimes the GH auto-closing back-fires ...

https://github.com/Microsoft/vscode/pull/41927#issue-164174861
> Stumbled on it trying to fix Microsoft/monaco-editor#574

**Comment by rcjsuen** [comment]
This is now working in the latest version of the Monaco Playground.
```TypeScript
monaco.languages.registerCodeActionProvider("json", {
    provideCodeActions: function(model, range, context, token) {
        return [
            {
                title: "action title"
            }
        ]
    }
});

monaco.editor.create(document.getElementById("container"), {
	value: "{ \"key\": \"value\" }",
	language: "json",
    lightbulb: {
        enabled: true
    }
});
```
